signingscript/src/signingscript/sign.py

```diff
@@ -1603,6 +1610,19 @@ async def apple_notarize_geckodriver(context, path, *args, **kwargs):
     return await _notarize_geckodriver(context, path, notarization_workdir)
 
 
+@time_async_function
+async def apple_notarize_openh264_plugin(context, path, *args, **kwargs):
+    """
+    Notarizes given geckodriver package using rcodesign.
```

Comment:
`s/geckodriver/openh264`

Also.. should we just move the `_notarize_openh264_plugin` logic inside this function?